test/unit/add-hooks-test.coffee

```diff
@@ -125,6 +125,77 @@ describe 'addHooks(runner, transactions, callback)', () ->
         globStub.sync.restore()
         done()
 
+    it 'should return files alphabetically sorted', (done) ->
+      addHooks runner, transactions, (err) ->
+        return done err if err
+
+        expected = [
+          'multifile_hooks.coffee',
+          'test2_hooks.js',
+          'test_hooks.coffee'
+        ]
+
+        actual = runner.hooks.configuration.options.hookfiles
+
+        actual.forEach (item, index) ->
+          assert.include item, expected[index]
```

Comment:
Addressed in https://github.com/apiaryio/dredd/pull/803/commits/c09efdb98fe2fe5ba294baefdd481b52e4aae413.